lib/teamcity/sqlServerCe/sqlServerCe_installation.txt

```diff
@@ -0,0 +1,15 @@
+
+Installation steps for local testing:
+
+1.  Create a project referencing System.Data.SqlServerCe.dll
+2.  Put following code in it, changing database file name and path as appropriate:
+	using (var en = new SqlCeEngine("DataSource=\"C:\\SqlCeData\\NHibernate.sdf\""))
+	{
+		en.CreateDatabase();
+	}
+3.  Run it.
```

Comment:
This is what TestDatabaseSetup does. But, it does not work properly for file-based DBs if you run the the tests from/with VS/R#.